.github/CONTRIBUTING.md

```diff
@@ -8,20 +8,55 @@ Afterwards, you can either contribute icons or code.
 ## Contributing icons
 To contribute icons, you only need an icon editor, a file explorer, a text editor, and a terminal window.
 
-### Icon guidelines
-See the below image for a summary of the icon guidelines. If you don't follow them, a team member will likely request changing the icons.
+### Creating Icons 1/5 — Artboard & Grid
```

Comment:
Instead of doing this format, try this instead:

```markdown
### Icon Guidelines
#### Artboard and Grid
Content

#### Details
Content

#### ...
Content
```

This will make the guidelines follow DRY principles. I'm uncertain whether you should keep the `#/5` text, maybe ask Chefski.

Reply:
Removed `#/5` text and fixed other formatting.